CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Fix redirect problem.
```

Comment:
This broke because the CWD when running under docker is different than when running under Flask dev server. I fixed the immediate problem, and then came back later and fixed tests.